v2/layout.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "id": "normal",
+    "description": "A regular one-sided card.",
+    "name": "Normal"
+  },
+  {
+    "id": "flip",
+    "description": "A card with a front and back side.",
+    "name": "Flip"
+  },
+  {
+    "id": "copy",
+    "description": "A card that is made up of multiple distinct cards with the same name.",
+    "name": "Copy"
```

Comment:
e.g. [Machilocation][(url)](https://netrunnerdb.com/en/card/14010) and other campaign cards